### PR TITLE
Fix #192 - Expose platform configuration file

### LINKED ISSUE #192: Add the ability to grab stripe configuration
PlatformServices should expose the tc-config the server was started with.

### PR DESCRIPTION
**Pull request by myronkscott**
expose the raw platform configuration file to PlatformService

### CODE REVIEW

File: standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java [hunk 1]
```diff
@@ -30,4 +31,8 @@
    * Dumps platform state into logs
    */
   void dumpPlatformState();
+  /**
+   * Provides the raw config file from server startup
+   */
+  InputStream getPlatformConfiguration();
```
Comment (ljacomet) [inline standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java]: Can't we make that a String that contains the content of the config file?

Comment (myronkscott) [inline standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java]: I'd rather have the downstream consumer do the conversion.  We can do it prior but that just means possibility messing up the character set.  I believe some consumers (jaxb) have the ability to detect things like that if the file is not in the platforms default. That being said, I'm not opposed to doing the conversion to the os default if that is what is truly desired.

Comment (ljacomet) [inline standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java]: Had not thought of the charset problem. Let's go with that, it is an easy thing to evolve should we decide to.

Comment (anthonydahanne) [inline standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java]: UTF-8 only ?

Comment (myronkscott) [inline standard-cluster-services/src/main/java/org/terracotta/monitoring/PlatformService.java]: Can you clarify?  Are you saying that we should mandate config file writers only use UTF-8?

**Pull request merged**